_includes/footer.html

```diff
@@ -1,21 +1,167 @@
-<footer role="contentinfo" id="site-footer">
-  <div class="container">
-    <div class="copyright pull-left">
-      <img src="{{ '/images/favicon-white.png' | relative_url }}" alt="{{ site.title }}" itemprop="image" width="16" height="16">
-      {{ site.time | date: '%Y' }} {{ site.owner["name"] }}
-    </div>
+<footer role="contentinfo" id="site-footer" class="syn-styles-supported">
+  <div class="syn-container">
+    <div class="syn-row syn-mb-32">
+      <div class="syn-col syn-col-lg-2 syn-col-md-3">
+        <div class="syn-mb-8">
+          <a href="/">
+            {% if site.enterprise %}
+            <img
+              v-else
```

Comment:
No need for the `v-else` here since we're outside the context of Vue.

Reply:
Removed!